src/OpenTok/OpenTok.php

```diff
@@ -385,6 +385,74 @@ public function listArchives($offset=0, $count=null)
         return new ArchiveList($archiveListData, array( 'client' => $this->client ));
     }
 
+    /**
+     * Initiate an outgoing SIP call
+     *
+     * @param string $sessionId TThe OpenTok SessionIdwhere the participant being called
+     * will join.
+     *
+     * @param string $token The token for conecting to an OpenTok session. This is the same
+     * as the one created by Session.generateToken.
+     *
+     * @param string $username Auth​: Username to be used in the SIP INVITE request for HTTP
+     * Digest authentication in case this is required by the Third Party SIP Platform.
+     *
+     * @param string $password Auth​: Password to be used in the SIP INVITE request for HTTP
+     * Digest authentication in case this is required by the Third Party SIP Platform.
+     *
+     * @param string $sipUrl The SIP Uri to be used as destination of the SIP Call initiated from
+     * OpenTok to the Third Party SIP Platform.
+     * If the SIP Uri contains a transport=tlsheader, the negotiation between TokBox and
+     * the SIP Endpoint will be done securely. Note that this will only apply to the negotiation
+     * itself, and not to the transmission of audio. If you also need the latter, please see the
+     * "secure" property.
+     * Example of secure call negotiation:
+     * "sip:access@thirparty.com;transport=tls"
+     * Example of insecure call negotiation:
+     * "sip:access@thirparty.com"
+     *
+     * @param array $options This array defines options for the token. This array includes the
+     * following keys, all of which are optional:
+     *
+     * <ul>
+     *
+     *    <li><code>'headers'</code> (string) &mdash; Headers​: Custom Headers to be added to the
+     *    SIP INVITE request initiated from OpenTok to the Third Party SIP Platform. All of this
+     *    custom headers must start with the "X-" prefix, or a Bad Request (400) will be thrown.</li>
+     *
+     *    <li><code>'secure'</code> (int) &mdash; Secure​: Boolean (trueor false) flag that indicates
+     *    whether the media must be transmitted encrypted or not.</li>
+     *
+     * </ul>
+     *
+     * @return SipCall The SipCall, which contains the ids of the Sip connection.
+     */
+    public function call($sessionId, $token, $sipUri, $username, $password, $options=array())
```

Comment:
i think we're going with `dial()` as the name of the API. we have a few weeks to really decide, but i just wanted to comment on where most of the consensus is right now.